jitbuilder/release/src/InliningRecFib.cpp

```diff
@@ -1,199 +0,0 @@
-/*******************************************************************************
```

Comment:
it would be better if these deleted files could be done as moves rather than as deletes and new files

Reply:
Git recognizes a delete+new as a move only if the files are "similar enough". I don't know of any way of changing the threshold, so I'm not sure this can be fixed :disappointed: .